index.html

```diff
@@ -39,17 +39,143 @@ <h3 class="connect">LET'S CONNECT</h3>
+        <div class="project-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul>
+          <li class="languages">HTML</li>
+          <li class="languages">CSS</li>
+          <li class="languages">JAVASCRIPT</li>
+        </ul>
+        <div>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>
+      <div class="projects">
+        <img src="./images/Snapshoot Portfolio (1).png" alt="">
+        <h2 class="works-title">TONIC</h2>
+        <p><span class="canopy">Canopy</span> <img class="canopy" src="./images/Counter.png" alt="Counter"> <span class="dev">   End Dev
+            <img src="./images/Counter.png" alt="Counter"> 2015</span></p>
+        <div class="project-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div>
+          <button class="languages">HTML</button>
+          <button class="languages">CSS</button>
+          <button class="languages">JAVASCRIPT</button>
+        </div>
+        <div>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>
+
+      <div class="projects">
+        <img src="./images/Snapshoot Portfolio (2).png" alt="">
+        <h2 class="works-title">TONIC</h2>
+        <p><span class="canopy">Canopy</span> <img src="./images/Counter.png" alt="Counter"> <span class="dev"> End Dev
+            <img src="./images/Counter.png" alt="Counter"> 2015</span></p>
+        <div class="project-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div>
+          <button class="languages">HTML</button>
+          <button class="languages">CSS</button>
+          <button class="languages">JAVASCRIPT</button>
+        </div>
+        <div>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>
+
+      <div class="projects">
+        <img src="./images/Snapshoot Portfolio(3).png" alt="">
+        <h2 class="works-title">TONIC</h2>
+        <p><span class="canopy">Canopy</span> <img src="./images/Counter.png" alt="Counter"> <span class="dev"> End Dev
+            <img src="./images/Counter.png" alt="Counter"> 2015</span></p>
+        <div class="project-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div>
+          <button class="languages">HTML</button>
+          <button class="languages">CSS</button>
+          <button class="languages">JAVASCRIPT</button>
+        </div>
+        <div>
+          <button class="see-project">See Project</Button>
+        </div>
+      </div>
+
+      <section class="about">
+        <div>
+          <div class="first-heading">
+            <h2>About Myself</h2>
+          </div>

```

Comment:
- Please kindly remove the border in the `About` as it is not present in the original design so as to stick to the given design as much as possible. 🤗

Your design
![image](https://github.com/MargaretTechworld/Portfolio/assets/78251927/ebdc82ea-a845-4b97-9f3a-11dda85e6b79)

Figma
![image](https://github.com/MargaretTechworld/Portfolio/assets/78251927/f8b28fd1-a6ad-4c3e-8a3c-1c2cd3dc4b3a)

- Kindly note the background color of the parent element 👀🛠 💪